We are not using MongoDB's Bazelisk wrapper, so disable this check.

diff '--color=auto' -ur a/bazel/toolchains/cc/mongo_linux/mongo_compiler_flags.bzl b/bazel/toolchains/cc/mongo_linux/mongo_compiler_flags.bzl
--- a/bazel/toolchains/cc/mongo_linux/mongo_compiler_flags.bzl	2026-06-15 18:37:04.000000000 -0400
+++ b/bazel/toolchains/cc/mongo_linux/mongo_compiler_flags.bzl	2026-06-16 14:22:58.517254532 -0400
@@ -68,8 +59,7 @@
 
 MONGO_LINUX_CC_COPTS = (
     MONGO_GLOBAL_INCLUDE_DIRECTORIES +
-    SYMBOL_ORDER_COPTS +
-    RUNNING_THROUGH_BAZELISK_CHECK
+    SYMBOL_ORDER_COPTS
 )
 
 MONGO_LINUX_CC_LINKFLAGS = (
